packages/react-scheduler/src/ReactScheduler.js

```diff
@@ -144,12 +184,8 @@ if (!ExecutionEnvironment.canUseDOM) {
       frameDeadlineObject.didTimeout = false;
     }
 
-    timeoutTime = -1;
-    const callback = scheduledRICCallback;
-    scheduledRICCallback = null;
-    if (callback !== null) {
-      callback(frameDeadlineObject);
-    }
+    scheduledCallbackConfig = null;
+    safelyCallScheduledCallback(callback, callbackId);
```

Comment:
Does this need to be a separate function call? I guess it could get inlined anyway so it probably doesn't matter.

Reply:
This does eventually get called in multiple places, and will be more complex once we implement error handling.